src/components/MatchRuleList.tsx

```diff
@@ -0,0 +1,279 @@
+import { useTranslation } from 'react-i18next'
+import { Translation, Budget } from '../types'
+import { useEffect, useState } from 'react'
+import {
+  ArrowLongDownIcon,
+  ArrowLongRightIcon,
+  CloudArrowUpIcon,
```

Comment:
We can remove this icon import thanks to https://github.com/envelope-zero/frontend/commit/340d8ce26b26787162f4e119f97ba399e63948a1

Reply:
3382e9c